### DISCUSSION TITLE
How is translation done depending on the language?

### DISCUSSION THREAD

**Discussion by ismailix** [ng-matero/ng-matero#652] [Q&A] [answered]
https://ng-matero.github.io/ng-matero/material/autocomplete Here are a few examples of how I can translate the texts on this page into English, Chinese or Japanese. Can you tell me with a few examples?

{{'dashboard.menu' | I translate}} but I get an error.

**Comment by nzbin** [comment, marked as answer]
```
{{'dashboard.menu' | I translate}} 
                     👆
{{'dashboard.menu' | translate}}
```
Please check the ngx-translate docs
https://github.com/ngx-translate/core?tab=readme-ov-file#5-use-the-service-the-pipe-or-the-directive

**Reply by ismailix** [reply]
TranslateModule the problem was fixed with the module
